### ISSUE TITLE
Relax first time contributor CI approval rules

### ISSUE THREAD

**Issue by danobi** [iovisor/bpftrace#2882]
Currently new contributors need their GHA runs approved before each run. This happens until they get a commit into the repo. It's kinda a bad contributor experience: https://matklad.github.io/2022/10/24/actions-permissions.html

What do we think about setting it to first time GH users requiring approvals?

**Comment by jordalgo** [comment]
What is the downside to relaxing this? Possible spam / over use of the CI?

**Comment by danobi** [comment]
I recall this appoval thing was added recently. Before anyone could trigger a job.

Downside must be for overuse. ANd if you use custom runners, perhaps malicious use

**Comment by jordalgo** [comment]
I guess it could be locked back down if that happens.

**Comment by viktormalik** [comment]
Agreed, let's relax. We've had a couple of first-time contributors recently and it's a bit annoying to keep starting the CI manually all the time.

**Comment by danobi** [comment]
Config updated
